src/snowflake/sqlalchemy/snowdialect.py

```diff
@@ -132,7 +132,11 @@ class SnowflakeDialect(default.DefaultDialect):
     #  unicode strings
     supports_unicode_statements = True
     supports_unicode_binds = True
-    returns_unicode_strings = String.RETURNS_UNICODE
+    # no longer used for SQLAlchemy 2.0, so any value works
+    # String.RETURNS_UNICODE for backwards compatibility with 1.4
+    returns_unicode_strings = (
+        String.RETURNS_UNICODE if hasattr(String, "RETURNS_UNICODE") else None
+    )
```

Comment:
This is not necessary in 2.0, but kept for backwards-compatibility. `String.RETURNS_UNICODE` doesn't exist in 2.0.